go.mod

```diff
@@ -67,6 +67,25 @@ require (
 	knative.dev/networking v0.0.0-20210914225408-69ad45454096
 )
 
+require (
+	github.com/StackExchange/wmi v1.2.1 // indirect
+	github.com/amenzhinsky/iothub v0.7.0
+	github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f // indirect
+	github.com/beeker1121/goque v2.1.0+incompatible // indirect
+	github.com/confluentinc/confluent-kafka-go v1.4.2 // indirect
+	github.com/gofrs/uuid v4.0.0+incompatible // indirect
+	github.com/inconshreveable/log15 v0.0.0-20201112154412-8562bdadbbac // indirect
+	github.com/kevinburke/go-types v0.0.0-20210723172823-2deba1f80ba7 // indirect
+	github.com/kevinburke/rest v0.0.0-20210506044642-5611499aa33c // indirect
+	github.com/sendgrid/rest v2.6.5+incompatible // indirect
+	github.com/shirou/gopsutil v3.21.8+incompatible // indirect
+	github.com/syndtr/goleveldb v1.0.0 // indirect
+	github.com/ttacon/builder v0.0.0-20170518171403-c099f663e1c2 // indirect
+	github.com/ttacon/libphonenumber v1.2.1 // indirect
+	golang.org/x/tools v0.1.6 // indirect
+	gopkg.in/sourcemap.v1 v1.0.5 // indirect
```

Comment:
Why do we need to move all these imports?

Reply:
I dont know i didnt touch this go mod tidy did it.

Reply:
I'm really starting to be convinced this wasn't generated by Go 1.17 🤔

Reply:
ill delete my go and re install

Reply:
lets see what happens

Reply:
yea.. this is what i get :/